support-frontend/assets/helpers/tracking/ophan.ts

```diff
@@ -74,7 +74,7 @@ type OphanABEvent = {
 	campaignCodes?: string[];
 };
 
-type OphanABPayload = Record<TestId, OphanABEvent>;
+type OphanABPayload = Record<string, OphanABEvent>;
```

Comment:
`TestId` was just an alias for `string` and I don't think it was adding anything